Firestore/Source/Local/FSTLocalViewChanges.mm

```diff
@@ -18,6 +18,7 @@
 
 #include <utility>
 
+#import "FSTTypes.h"
```

Comment:
Shouldn't this be Firestore/Source/Core/FSTTypes.h? (i.e. full path)

But never mind that. You can just remove this line entirely, since the header already includes it.

Reply:
Fixed